src/Core/composer.json

```diff
@@ -25,6 +25,9 @@
         "php": ">=7.4.0",
         "cakephp/utility": "^4.0"
     },
+    "provide": {
+        "psr/http-server-handler-implementation": "^1.0"
```

Comment:
This doesn't seem right.

Reply:
Ah, it's cause `HttpApplicationInterface` in this package extends `RequestHandlerInterface`.

Reply:
Yeah I mainly looked for any implementations and fixed those packages

Reply:
I believe an implementation requires a class, not just an interface.